CONTRIBUTING.md

```diff
@@ -124,6 +124,13 @@ Also Bootstrap mentions http://getbootstrap.com/getting-started/#examples
 as examples of things you can do, but they are not part of the core library,
 therefore this project is the wrong place to implement them.
 
+## Notes for lodash functions usage in the code
+
+You can use `lodash` but keep it to things where it actually needs it, i.e. don't use `lodash`'s `forEach` when `Array.prototype.forEach` is fine.
```

Comment:
That's an example, but should be consider including a "whitelist" of when to prefer native implementations? `Array.prototype.sort`/`map`/`reduce`/ etc.